anime/components/awardspage/ActualVoting.vue

```diff
@@ -195,7 +196,7 @@ function select(anime: String) {
   nominee.value = click.outerText;
 
   nomineeBox.value.forEach((box) => {
-    box.style.background = "#252525";
+    box.style.background = "var(--bg-secondary)";
   });
   if (click.className == "nominee-box") {
     click.style.background = "var(--primary)";
```

Comment:
btw you can bind classes to variables so

```javascript

const isActive = ref(true)
const hasError = ref(false)

<div
  :class="{ active: isActive, 'text-danger': hasError }"
></div>
```

might be helpful

Reply:
I tried but it keeps turning the class active for all of them. How do I make the class active only for the box I click on?